src/MicrosoftTeams.ts

```diff
@@ -684,6 +684,26 @@ namespace microsoftTeams {
     sendMessageRequest(parentWindow, "openFilePreview", params);
   }
 
+  /**
+   * @private
+   * Hide from docs.
+   * ------
+   * Sideload an App-manifest directly to a team.
```

Comment:
Wouldn't this allow for apps to be sideloaded both personally or into a team? If so, please update the comment.